### PR TITLE
Avoid redirecting all web-onboarding pages to info

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

We want to still use connect payment page to update payment methods

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Review by guilhermespopolin — COMMENTED** [review]
Just left a comment. It seems one definition is missing

File: apps/store/next.config.js [hunk 1]
```diff
@@ -94,25 +94,49 @@ module.exports = withBundleAnalyzer({
     return process.env.NEXT_PUBLIC_FEATURE_HOMECOMING === 'true'
       ? [
           {
-            source: '/no/new-member/:slug*',
+            source: '/no/new-member',
```
Comment (alebedev) [inline apps/store/next.config.js:97]: Can we use regexp to have singe matching source for each country+locale redirect to info page?

**Review by alebedev — APPROVED** [review]
Looks good

**Review by guilhermespopolin — APPROVED** [review]
Nice :)

**Pull request merged**